### ISSUE TITLE
mission statement

### ISSUE THREAD

**Issue by agboch** [troettge/RN-Norway#5]
Terms of reference draft (including Vision, Purpose, Scope and Organization) is [here ](https://docs.google.com/document/d/1xKdIuNIyy0XGunV22mNRSHa_oBSdL3sPcVQxYljY6iw/edit?usp=sharing)- still very much work in progress. 
Tagging everybody for visibility: @dsquintana @troettge @julienmayor

**Comment by julienmayor** [comment]
Let me know if you want a hand doing this!

**Comment by agboch** [comment]
The current draft is adapted mostly from the Finnish Terms of Reference, but especially for the RN organization part, different networks do it differently - so lots to discuss and decide on before we can settle on any details there, might have to be rewritten substantially. I think everybody should read (or at least take a look at) other networks terms of reference.

**Comment by troettge** [comment]
Should we set a deadline for comments so that we can upload a first version to the website? Maybe Thursday the 18th and then a last discussion about residual issues at our next meeting?

**Comment by agboch** [comment]
Sounds good to me!

**Comment by troettge** [comment]
> Should we set a deadline for comments so that we can upload a first version to the website? Maybe Thursday the 18th and then a last discussion about residual issues at our next meeting?

bump for all to give final feedback till Friday @julienmayor  @dsquintana.

**Comment by julienmayor** [comment]
thanks for the reminder - although my suggestion is that the document should end with "organisation" where we say we start with our core group for now and that a mature structure will emerge in the near future. Whether this will be with a steering group + local nodes, etc will, precisely, be decided then.

**Comment by agboch** [comment]
We discussed this in the last meeting and we decided that we will leave the organization part with a clear statement that it is preliminary and it might change. But I think we should quickly revisit this discussion tomorrow since you have a clear stance on this @julienmayor

**Comment by julienmayor** [comment]
I've edited the text to put the description of the future structure... in the future tense. Feel free to modify at your will!

Something that I feel hasn't be discussed there (and is contentious); should we ask that all NORRN documentation, meetings, communication (oral and written) is in English? For the sake of inclusivity (for new-comers in Norway), I think this would be great, so that they aren't isolated until they speak Norwegian - and so that it does not become an issue of contention down the line - e.g., people can refer to some official guidelines. It can also backfire, as some people feel strongly that Norwegian should be used more in academic settings - so I can also see the argument for leaving a void here... @troettge @agboch @dsquintana any thoughts?

**Comment by troettge** [comment]
Well, you know my stance on this. English is clearly the more inclusive language and I don't see any convincing reasons not to.

**Comment by agboch** [comment]
I think that everything should be in English, also because we are a part of a global network. 

However, I also think that it would be nice to be able to link the Norwegian version of the Terms of Reference and the Code of Conduct for those that don't feel as comfortable reading these types of docs in English (and also because those state the official goals and ways of conduct for the network and it would be nice to have them in both languages since it's a Norwegian leg of the organization). What do others think?

**Comment by sirileknes** [comment]
I agree English is the best option for the reasons you guys state above. I do not think we need to prioritise translating anything to Norwegian.

**Comment by sirileknes** [comment]
I think the mission statement looks great

**Comment by troettge** [comment]
Nothing to add from my side either, I only added the sentence: "The initiating members will be temporarily steering the network until a governing structure has been established." to the blue disclaimer to go well with Siri's changes to "steering committee" through the text. @agboch give me a ping, when you think I should update the website regarding the ToR.

**Comment by agboch** [comment]
@troettge I read through and cleaned up the ToR  document (but I archived the version with the marked parts that we will maybe want to change later). If @dsquintana and @julienmayor also accept that last version, it's free to go up on the website!

**Comment by troettge** [comment]
@julienmayor @dsquintana give us a :thumbsup: for the ToR when you have time.

**Comment by dsquintana** [comment]
I'm happy with ToR

**Comment by julienmayor** [comment]
All good!
